aio/closure/webpack.config.js

```diff
@@ -69,6 +69,7 @@ module.exports = {
     'styles': [
       '../node_modules/material-design-icons/iconfont/material-icons.css',
       '../node_modules/roboto-fontface/css/roboto/roboto-fontface.css',
+      '../node_modules/nvd3/build/nv.d3.css',
```

Comment:
What is this change? Files in the `closure` dir are not actually used anywhere. They are leftovers from old builds kept until we finish everything in case they are needed.

Reply:
This is the CSS I was talking about on Slack, it's not included by default. I was not aware of how are CSS files included from `node_modules` in kube-dashboard. I looked for similar includes and found this in webpack config, added the statement and it worked. If you can suggest any other method to include this CSS, I'll try that. The goal is just to include this CSS somehow in the bundle.

Reply:
I mean that this should only be added to the `angular.json` file. This `webpack.config.js` file is basically not used at all.

Reply:
Okay, my bad. It works with changes to `angular.json` only.